src/components/dev-hub/breadcrumb-schema.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { useSiteMetadata } from '../../hooks/use-site-metadata';
```

Comment:
```suggestion
import { useSiteMetadata } from '~hooks/use-site-metadata';
```

We have an alias for this subdirectory, so let's update that here